conf.py

```diff
@@ -28,56 +33,70 @@
         'versions': [
             {
                 'product': '0.23.0',
+                'java-base': '11',
                 'authorizer': '0.2.0',
             },
             {
                 'product': '24.0.0',
+                'java-base': '11',
                 'authorizer': '0.4.0',
             }
         ]
     },
     {
         'name': 'hadoop',
-        'versions': [{'product': '3.2.2'}, {'product': '3.3.1'}, {'product': '3.3.3'}, {'product': '3.3.4'}],
+        'versions': [
+            {'product': '3.2.2', 'java-base': '11'},
+            {'product': '3.3.1', 'java-base': '11'},
+            {'product': '3.3.3', 'java-base': '11'},
+            {'product': '3.3.4', 'java-base': '11'},
+        ],
     },
     {
         'name': 'hbase',
         'versions': [
             {
                 'product': '2.4.6',
+                'java-base': '11',
                 'phoenix': '2.4-5.1.2',
             },
             {
                 'product': '2.4.8',
+                'java-base': '11',
                 'phoenix': '2.4-5.1.2',
             },
             {
                 'product': '2.4.9',
+                'java-base': '11',
                 'phoenix': '2.4-5.1.2',
             },
             {
                 'product': '2.4.11',
+                'java-base': '11',
                 'phoenix': '2.4-5.1.2',
             },
             {
                 'product': '2.4.12',
+                'java-base': '11',
                 'phoenix': '2.4-5.1.2',
             },
         ]
     },
     {
         'name': 'hive',
         'versions': [
-            {
-                'product': '2.3.9',
-                'hadoop': '2.10.1',
-                'jackson_dataformat_xml': '2.7.9',
-                'aws_java_sdk_bundle': '1.11.271',
-                'azure_storage': '7.0.1',
-                'azure_keyvault_core': '1.0.0',
-            },
+            # Hadoop 2.10.1 is no longer supported
+            # {
+            #     'product': '2.3.9',
+            #     'hadoop': '2.10.1',
+            #     'jackson_dataformat_xml': '2.7.9',
+            #     'aws_java_sdk_bundle': '1.11.271',
+            #     'azure_storage': '7.0.1',
+            #     'azure_keyvault_core': '1.0.0',
+            # },
```

Comment:
I assume that it's an oversight that we still have this in the build file, are we OK dropping it?

Reply:
Reenabled in cc8e73a605db1450a6b60f7f3d7d2deed63aaeb1.